Makefile

```
@@ -174,15 +174,11 @@ clean:
 	@echo "$$(tput bold)Clean rustdoc" && rm -rf doc/rustdoc
 	@echo "$$(tput bold)Clean ci-artifacts" && rm -rf tools/ci-artifacts
 
-.PHONY: fmt format
-fmt format: tools/.format_fresh
-	$(call banner,Formatting complete)
-
-# Get a list of all rust source files (everything fmt operates on)
-$(eval RUST_FILES_IN_TREE := $(shell (git ls-files | grep '\.rs$$') || find . -type f -name '*.rs'))
-tools/.format_fresh: $(RUST_FILES_IN_TREE)
-	@./tools/run_cargo_fmt.sh $(TOCK_FORMAT_MODE)
-	@touch tools/.format_fresh
+.PHONY: fmt format-check
+fmt format-check:
```

```suggestion
format-check:
```

or, if we're committed to keeping a shorthand `fmt-chk`, but `fmt` alone is a verb that formats IMHO, which this no longer does IIUC

---

Right, I completely missed this shorthand. Happy to update.